### ISSUE TITLE
We should define/recommend splash images

### ISSUE THREAD

**Issue by joshmarinacci** [immersive-web/proposals#22]
There will be some interfaces that collect a group of WebVR links into a catalog like interface, meaning a group of larger images. The really is no standard for this today, but we could at least encourage people to use one of the Open Graph protocol elements, similar to what Twitter does.

http://ogp.me/

https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary-card-with-large-image.html

**Comment by kfarr** [comment]
Good example of an interface like this (both the menu and title cards) from Supermedium: https://www.youtube.com/watch?v=QZVSVW5_taw

**Comment by cwilso** [comment]
This is a generic web page ask, not specific to XR, correct?  This seems covered by Web App Manifests (https://www.w3.org/TR/appmanifest/), e.g. app icons and/or screenshots.

**Comment by cvan** [comment]
> This is a generic web page ask, not specific to XR, correct? This seems covered by Web App Manifests (https://www.w3.org/TR/appmanifest/), e.g. app icons and/or screenshots.

yes, I totally recommend leveraging the [Web-App Manifest](https://w3c.github.io/manifest/).

there's some interesting background info. in the section about the [`background_color` key](https://w3c.github.io/manifest/#background_color-member):

> The `background_color` member is only meant to improve the user experience while a web application is loading and MUST NOT be used by the user agent as the background color when the web application's stylesheet is available.

it's totally okay to extend the manifest format without needing the spec to change. see [the Extensibility section](https://w3c.github.io/manifest/#extensibility) in the spec.

so you could imagine a manifest containing a key like this one:

```json
{
  "xr_splash_images": [
    {
      "src": "/img/splash.png",
      "type": "image/png"
    }
  ]
}
```

> That raises a good point, though... do we want to be considering how XR plays with the display-mode option? https://www.w3.org/TR/appmanifest/#display-modes
> Perhaps that's a different issue to be filed though?

:+1: yep, I still like the idea of leveraging `display-mode`.

I saw that you, @kearwood, and others were involved in the discussion in the CSS issue, w3c/csswg-drafts#2723. I see a PR was also merged: w3c/csswg-drafts#2719

**Comment by TrevorFSmith** [comment]
It seems like forward motion on this topic has stalled so I'm going to close this Issue. If someone has a plan for how to move it forward then ping me and we can re-open it.
